src/commands/database-rules-setlabels.ts

```diff
@@ -0,0 +1,23 @@
+import * as Command from "../command";
+import * as logger from "../logger";
+import * as requireInstance from "../requireInstance";
+import * as requirePermissions from "../requirePermissions";
+import * as metadata from "../database/metadata";
+
+export default new Command("database:rules:setlabels")
+  .description("list realtime database rulesets")
+  .option(
+    "--instance <instance>",
+    "use the database <instance>.firebaseio.com (if omitted, uses default database instance)"
+  )
+  .option("--stable <ruleset_id>", "mark the given ruleset id as 'stable'")
+  .option("--canary <ruleset_id>", "mark the given ruleset id as 'canary'")
```

Comment:
Please document the behavior if only one is specified. Please reject with an error / document behavior when none is specified.

Reply:
Made the current invariants more clear.

This will be a useful place to get API feedback.